backend/de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/allocation/spi/impl/AggregateHUTrxListener.java

```diff
@@ -70,12 +71,13 @@
  */
 public class AggregateHUTrxListener implements IHUTrxListener
 {
-	public static final AggregateHUTrxListener INSTANCE = new AggregateHUTrxListener();
+	public static final AggregateHUTrxListener INSTANCE = new AggregateHUTrxListener(SpringContextHolder.instance.getBean(HUQRCodesService.class));
```

Comment:
pls pls pls don't access the Spring context in a static way

Reply:
yeah, ofc not, just wanted to see if it fails to reference it back here: https://github.com/metasfresh/metasfresh/pull/20046#discussion_r1961349017

Reply:
as I was sure that's why we did it